### PR TITLE
Add lobby ready count to game ticker info during pregame

### PR DESCRIPTION
**Pull request by theashtronaut**
## About the PR 

During PreGame the lobby info will display the current readied player count. The ready count disappears if it is no longer PreGame (round starts or is post round).

**Screenshots**

https://user-images.githubusercontent.com/112137107/197605583-98335aa7-0394-47a4-b2da-e915f59aea2f.mp4

~~I couldn't find a fluent function to change are -> is for when the number is exactly 1, if that exists lmk and I can tweak it~~
this is fixed now thanks to ShadowCommander!

**Changelog**

:cl: ashtronaut
- add: Added a current readied player count to the lobby info during the pre-game phase.

### CODE REVIEW

File: Resources/Locale/en-US/game-ticking/game-ticker.ftl [hunk 1]
```diff
@@ -13,6 +13,12 @@ game-ticker-get-info-text = Hi and welcome to [color=white]Space Station 14![/co
                             The current map is: [color=white]{$mapName}[/color]
                             The current game mode is: [color=white]{$gmTitle}[/color]
                             >[color=yellow]{$desc}[/color]
+game-ticker-get-info-preround-text = Hi and welcome to [color=white]Space Station 14![/color]
+                            The current round is: [color=white]#{$roundId}[/color]
+                            The current player count is: [color=white]{$playerCount}[/color] ([color=white]{$readyCount}[/color] are ready)
```
Comment (ShadowCommander) [inline Resources/Locale/en-US/game-ticking/game-ticker.ftl:18]:
```suggestion
                            The current player count is: [color=white]{$playerCount}[/color] ([color=white]{$readyCount}[/color] {$readyCount ->
                                [one] is
                                *[other] are
                            } ready)
```

**Review by ShadowCommander — APPROVED** [review]

File: Resources/Locale/en-US/game-ticking/game-ticker.ftl (hunk 1 above)
Comment (theashtronaut) [reply Resources/Locale/en-US/game-ticking/game-ticker.ftl]: amazing, thank you

**Comment by theashtronaut** [comment]
updated to fix the grammar so now it says is for 1 and are for other amounts (thanks ShadowCommander! :heart: )

File: Content.Server/GameTicking/GameTicker.Lobby.cs [hunk 2]
```diff
@@ -41,11 +42,14 @@ private string GetInfoText()
...
             var playerCount = $"{_playerManager.PlayerCount}";
+            var readyCount = _playerGameStatuses.Values.Count(x => x == PlayerGameStatus.ReadyToPlay);
             var map = _gameMapManager.GetSelectedMap();
             var mapName = map?.MapName ?? Loc.GetString("game-ticker-no-map-selected");
             var gmTitle = Loc.GetString(Preset.ModeTitle);
             var desc = Loc.GetString(Preset.Description);
-            return Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc));
+            return RunLevel == GameRunLevel.PreRoundLobby
+                ? Loc.GetString("game-ticker-get-info-preround-text",("roundId", RoundId), ("playerCount", playerCount),("readyCount", readyCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc))
+                : Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc));
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/GameTicker.Lobby.cs:52]: Surely could just pass in the tuples to the function and avoid duplicating them all??

**Review by metalgearsloth — DISMISSED** [review]

File: Content.Server/GameTicking/GameTicker.Lobby.cs (hunk 2 above)
Comment (theashtronaut) [reply Content.Server/GameTicking/GameTicker.Lobby.cs]: For some reason I thought passing in arguments that don't exist would trigger errors. Like passing in ready count for game-ticker-get-info-text with no variable for ready count would trigger an error. Seems that isn't the case, very good to know.
simplified/fixed this

**Review by rolfero — APPROVED** [review]

**Comment by theashtronaut** [comment]
updated to resolve merge conflict, all tests passed locally, functionality regarding map name mirrors how it is working on master currently, functionality this adds remains functional

(i didn't feel like updating the video tonight)

**Pull request merged**